Packet++/header/GtpLayer.h

```diff
@@ -211,243 +211,244 @@ namespace pcpp
 	 * @class GtpV1Layer
```

Comment:
it is very difficult to keep track of the changes you've done in this file. It seems most of them are indentation fixes but I'm not sure. Maybe it'd be easier to make these changes in phases: first start with indentation fixes only and then move to more fixes when needed

Reply:
I have returned back the original file and made small number of changes.